gammapy/background/tests/test_fov.py

```diff
@@ -1,2 +1,110 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function, division
+
+from gammapy.background import fill_acceptance_image
+from gammapy.image import make_empty_image
+
+from astropy.io import fits
+from astropy.coordinates import SkyCoord, Angle
+from astropy.wcs import WCS
+from astropy.wcs.utils import pixel_to_skycoord
+import astropy.units as u
+from astropy.units.quantity import Quantity
+
+import numpy as np
+from scipy.integrate import quad
+
+def radial_gaussian_2D(r, sig):
+    """Radially symmetric Gaussian function for emulating acceptance curve.
+
+    Please give parameters in radians.
+
+    TODO: I can't define parameters as `~astropy.coordinates.Angle` because it conflicts with the integral call afterwards!!!
+
+    Parameters
+    ----------
+    r : `~numpy.ndarray`
+    Radial coordinate.
+    sig : float
+    Gaussian width.
+
+    Returns
+    -------
+    val: `~numpy.ndarray`
+    Gaussian evaluated at the requested radii.
+    """
+    ##norm = 1./(sig * np.sqrt(2. * np.pi))
+    ##norm *= 2. #due to definition only for r>=0
+    ##norm /= (2.*np.pi) #due to radial symmetry
+    norm = 1.
+    val = norm * np.exp(-np.power(r, 2.) / (2. * np.power(sig, 2.)))
+    return val
+
+
+def test_fill_acceptance_image():
+
+    #create empty image
```

Comment:
Comments should only be used where they add extra value and things can't be expressed clearly in code.
So remove this `#create empty image` comment because it doesn't add extra info to the `image = make_empty_image()` code.